docs/deprecation-policy.md

```diff
@@ -0,0 +1,186 @@
+---
+assignees:
+- bgrant0607
+- lavalamp
+- thockin
+
+---
+
+# Kubernetes Deprecation Policy
+
+Kubernetes is a large system with many components and many contributors.  As
+with any such software, the feature set naturally evolves over time.  Sometimes
+we realize that something we have done in the past, whether that is some API, a
+flag, or even a whole feature, needs to be replaced or removed.  Kubernetes is
+committed to not breaking users, so we can’t just make breaking changes.  This
+is when deprecation is triggered.
+
+This document details the policies we hold on deprecation for various facets of
+the system.
+
+## Deprecating parts of the API
+
+Since Kubernetes is an API-driven system, it’s not surprising that the API
```

Comment:
"We/our" construct.

"Since Kubernetes is an API-driven system, the API has evolved over time to reflect an evolving understanding of the problem space. The Kubernetes API consists of several "API groups", and each API group is independently versioned."